lib/pg_column_byte_packer.rb

```diff
@@ -84,6 +122,14 @@ def self.ordering_key_for_column(connection:, name:, sql_type:, type_schema: nil
             4
           end
         end
+      when /\Abit varying\(\d+\)/
```

Comment:
This is an interesting case. I do wonder if it's worth special casing. I guess the code isn't too bad once you understand the reasoning. Might be worth a comment in the code to explain it.

Reply:
I think I'll add a comment separately since it should also apply to `character varying` above.